index.js

```diff
@@ -140,7 +144,7 @@ function copyNodeModules(srcDir, dstDir, opts, callback)
 
     if (!callback)
     {
-        g_opts = {srcDir: srcDir, dstDir: dstDir, devDependencies: false};
+        g_opts = {srcDir: srcDir, dstDir: dstDir, devDependencies: false, filter: null};
```

Comment:
Probably, you don't need to set `filter` option to `null` by default as it's already undefined.